Python/bytecodes.c

```diff
@@ -2797,18 +3085,19 @@ dummy_func(
             _ITER_JUMP_RANGE +
             _ITER_NEXT_RANGE;
 
-        op(_FOR_ITER_GEN_FRAME, (iter -- iter, gen_frame: _PyInterpreterFrame*)) {
-            PyGenObject *gen = (PyGenObject *)iter;
+        op(_FOR_ITER_GEN_FRAME, (iter -- iter, gen_frame)) {
```

Comment:
You'll need to change the code generator to emit 
```C
    stack_pointer[0].bits = (uintptr_t)gen_frame;
    gen_frame = (_PyInterpreterFrame*)stack_pointer[0].bits;
```
instead of 
```
     stack_pointer[0] = (PyObject *)gen_frame;
     gen_frame = (_PyInterpreterFrame*)stack_pointer[0];
```